scripts/build.sh

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env bash
+
+linux() {
+  echo Running on linux
+
+  sudo ./scripts/uni-get-dependencies.sh
+  ./scripts/check-dependencies.sh
+
+  qmake openscad.pro
+  make
+  sudo make install
+}
+
+mac() {
+  echo Running on macOS
+
+  if  [ -x "$(command -v brew)" ]; then
+    echo 'Looks like Homebrew is installed. Using Homebrew.' >&2
+    ./scripts/macosx-build-homebrew.sh
+
+    qmake openscad.pro
+    make
+    sudo make install
+  fi
+}
+
+
+# pull MCAD
+if ! [ -x "$(command -v git)" ]; then
+  echo 'Error: git is not installed.' >&2
+  exit 1
+fi
+cd ..
```

Comment:
It looks like this script must be run using the `scripts/` folder as CWD, but then cd's out of that folder.
This isn't really a very common behavior. Please consider another strategy for running this, e.g. `SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"`